config/jobs/kubeflow/kubeflow-presubmits.yaml

```diff
@@ -83,6 +83,19 @@ presubmits:
       testgrid-dashboards: sig-big-data
       description: Presubmits for kubeflow/website.
       testgrid-num-columns-recent: '30'
+  - name: kubeflow-website-link-check
```

Comment:
Is there a way to mark it as non-blocking so we can verify that is working correctly before we start blocking presubmits?

Reply:
Yes, per the [docs](https://github.com/kubernetes/test-infra/blob/master/prow/jobs.md) I set `always_run: false` and `skip_report: true`. If the test is working and helpful we can flip those later.

Reply:
Why set always_run to false? Is there a way to always run it and surface the result to the user without having failures be blocking?

Reply:
@scottilee. I think the field we want here is `optional: true`. This will allow the job to automatically and appear in the PR context, but make it non-blocking.

Reply:
https://github.com/kubernetes/test-infra/blob/860d1bd5012fae1ae7330e2b59806763ce2b430e/prow/config/jobs.go#L136-L137

Reply:
Great thanks